### PR TITLE
Introduce `enable_anbox_ime` feature flag

### PR DESCRIPTION
**Pull request by adglkh**

### CODE REVIEW

File: reference/ams-configuration.md [hunk 1]
```diff
@@ -56,6 +56,14 @@ The Android virtual keyboard is disabled by default but can be enabled with the
 
 For the feature to be considered, applications must be manually updated, because changes to allow the feature to work are only applied during the [application bootstrap process](https://discourse.ubuntu.com/t/managing-applications/17760#bootstrap).
 
+#### Client-Side Virtual Keyboard
+
+*since 1.11.0*
+
+The client-side virtual keyboard is disabled by default but can be enabled with the `enable_anbox_ime` feature flag. It requires the client application to embed [AnboxWebView](https://discourse.ubuntu.com/t/integrate-a-client-side-virtual-keyboard) which interacts with the client-side virtual keyboard for text editing and sends them to Android container.
```
Comment (ru-fu) [inline reference/ams-configuration.md:63]:
```suggestion
The client-side virtual keyboard is disabled by default but can be enabled with the `enable_anbox_ime` feature flag. It requires the client application to embed [AnboxWebView](https://discourse.ubuntu.com/t/integrate-a-client-side-virtual-keyboard), which interacts with the client-side virtual keyboard for text editing and sends the text to the Android container.
```

**Review by ru-fu — CHANGES_REQUESTED** [review]

File: howto/stream/client-side-keyboard.md [hunk 2]
```diff
@@ -66,6 +66,10 @@ Adding the following JavaScript code snippet builds up a communication tunnel be
 
    ```
 
+[note type="information" status="Note"]
+To enable the application running in the Android container can receive the texts sent from the client-side virtual keyboard, the feature `enable_anbox_ime` must be declared in the application manifest file when creating an application.
```
[the remark this answers is not in the document]
Comment (adglkh) [reply howto/stream/client-side-keyboard.md:70]: yeah, not ideal. 
Maybe, after 
```
For the complete implementation details, refer to the `enhanced_webview_streaming` example in the `example/android` folder.
``` and before the  ` 1. Import the AAR library`
How do you think?

File: reference/ams-configuration.md (hunk 1 above)
Comment (adglkh) [reply reference/ams-configuration.md:63]: +1, Will fix.

File: howto/stream/client-side-keyboard.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ru-fu) [reply howto/stream/client-side-keyboard.md:70]: Yes, that's another place I was thinking about. Sounds good! :+1:

**Review by ru-fu — APPROVED** [review]
Looks good!

**Pull request merged**